packages/php-wasm/compile/base-image/Dockerfile

```diff
@@ -38,8 +38,8 @@ RUN set -euxo pipefail;\
 # the build take forever on Apple Silicon.
 RUN ln -s /usr/bin/python3 /usr/bin/python
 RUN git clone https://github.com/emscripten-core/emsdk.git && \
-    ./emsdk/emsdk install 3.1.43 && \
-    /root/emsdk/emsdk activate 3.1.43
+    ./emsdk/emsdk install 3.1.60 && \
+    /root/emsdk/emsdk activate 3.1.60
```

Comment:
@fluiddot, you might have already noticed, but wanted to highlight that `3.1.61` was [released recently](https://github.com/emscripten-core/emscripten/commit/f080b30d662f81366899a338e0ce7b76ea136586). Do you seen any blockers to bumping this PR to `3.1.61` or do you think that should be done separately, after this one's merged?

Reply:
> [...] Do you seen any blockers to bumping this PR to `3.1.61` or do you think that should be done separately, after this one's merged?

Hey @SiobhyB 👋 ! No, I don't see any blockers. I'd like first to solve the issues we spotted in CI jobs, but after that, we could bump the version and recompile everything again.